docs/README.md

```diff
@@ -0,0 +1,41 @@
+## Kanister Documentation
+
+This folder contains the source of the Kanister documentation at
+https://docs.kanister.io
+
+## Building The Source
+
+The content are written in the
+[reStructuredText](https://docutils.sourceforge.io/rst.html) format.
```

Comment:
```suggestion
The content is written in the
[reStructuredText](https://docutils.sourceforge.io/rst.html) format.
```
[nit]

Reply:
```suggestion
The contents are written in the
[reStructuredText](https://docutils.sourceforge.io/rst.html) format.
```
@ihcsim We should probably do one of these changes.

Reply:
ah sorry, i missed that one. Will fix.